routes/handlers/facebook.js

```diff
@@ -1,9 +1,14 @@
 const Facebook = require('../../lib/facebook');
 
 module.exports.events = {
-  handler: function (request, reply) {
-    Facebook.getEvents(function(err, res) {
-      return reply(res).type('application/json');
-    });
-  }
+    handler: function (request, reply) {
+        Facebook.getEvents(function (err, res) {
+            return reply(res);
+        });
+    },
+    plugins: {
+        hal: {
```

Comment:
actually, I don't think this was a change I made.  I think it came from a pull from this master which probably came from the work on OSDI facebook events.  But let me double check to make sure.